Dockerfile

```diff
@@ -6,17 +6,13 @@ ARG model_files=models.tar.gz
 
 # Get model archive and unzip it to assets folder
 RUN wget -nv ${model_bucket}/${model_files} --output-document=/workspace/assets/${model_files} --show-progress --progress=bar:force:noscroll
-RUN tar -x -C assets/ -f assets/${model_files} -v
+RUN tar -x -C assets/ -f assets/${model_files} -v && rm assets/${model_files}
 
 # Install dependencies for wave file encoding
 RUN apt-get update && apt-get install -y ffmpeg && rm -rf /var/lib/apt/lists/*
```

Comment:
I don't think we need `ffmpeg` anymore since mp3 support is not included?

Reply:
Thanks for catching that